dist/index.d.ts

```diff
@@ -125,5 +125,25 @@ export declare class ObiexClient {
         maximumDecimalPlaces: number;
     }>;
     getOrCreateWallet(currencyCode: string): Promise<Wallet>;
+    /**
+     *
+     * @param payload BankDepositRequest
+     * @returns
+     */
+    depositNaira({ merchantCode, amount }: BankDepositRequest): Promise<any>;
+    /**
+     *
+     * @param reference string
+     * @returns
+     */
+    verifyNairaDeposit(reference: string): Promise<any>;
+    /**
+     *
+     * @param reference string
+     * @returns
+     */
+    verifyNairaWithdrawal(reference: string): Promise<any>;
 }
 export { ServerError } from "./errors/server";
+export { TransactionCategory } from "./enums/TransactionCategory";
+export { Currency, Network, Options, Quote, Response, TradePair, BankAccountPayout, CryptoAccountPayout, Wallet, FiatMerchant, Banks, BankDepositRequest, } from "./types";
```

Comment:
This should also be formatted per line.

Reply:
hmmm ok I will see if I format files in dist folder

Reply:
Oh. It's dist. Lol. Don't worry

Reply:
ok done